sound/soc/sof/ipc4-topology.h

```diff
@@ -363,4 +363,39 @@ struct sof_ipc4_process {
 	bool is_loadable;
 };
 
+/**
+ * struct sof_ipc4_pin_format - widgets pin format
+ * @pin_index: pin index for the widget
+ * @ibs: input buffer size for input pin
+ * @obs: output buffer size for output pin
+ * @audio_fmt: audio format for the pin
+ */
+struct sof_ipc4_pin_format {
+	u32 pin_index;
+	union {
+		u32 ibs;
+		u32 obs;
+	};
```

Comment:
single `u32 iobs` instead of the union?

Reply:
> SOF loadable module support

let's call it `SOF loadable module instance initialization support`

Reply:
is it applies to IPC3 as well, I don't think so, this is "IPC4 loadable module instance initialization support"